### PR TITLE
Update 68

### PR DESCRIPTION
**Pull request by stringhandler**

### CODE REVIEW

File: _updates/2022-01-13-update-68.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+title: Tari Developer Update
+class: subpage
+---
+
+### New Testnet 26 Jan 2022
+
+A quick reminder that the Dibbler testnet will be starting on 26 Jan 2022. This testnet includes functionality
+in the base layer which will enable sidechains such as the digital assets layer. The development of the 
+digital assets layer is in its very early stages but you can start looking at it already in the development branch. Be warned though, the APIs are 
+not final and are subject to change dramatic and without notice.
```
Comment (delta1) [inline _updates/2022-01-13-update-68.md]:
```suggestion
not final and are subject to change dramatically and without notice. 
```

File: _updates/2022-01-13-update-68.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+A quick reminder that the Dibbler testnet will be starting on 26 Jan 2022. This testnet includes functionality
+in the base layer which will enable sidechains such as the digital assets layer. The development of the 
+digital assets layer is in its very early stages but you can start looking at it already in the development branch. Be warned though, the APIs are 
+not final and are subject to change dramatic and without notice. 
+
+### Covenants
+
+Other features to look forward to in Dibbler are covenants. Covenants are an extension to standard Mimblewimble which allow 
+the creator of a UTXO to dictate the rules about *how* it is spent. One main use case is to ensure that UTXO's that represent
```
Comment (delta1) [inline _updates/2022-01-13-update-68.md]:
```suggestion
the creator of a UTXO to dictate the rules about *how* it is spent. One main use case is to ensure that UTXOs that represent
```

**Review by delta1 — APPROVED** [review]
Looks good, maybe add links to discord etc

**Review by CjS77 — APPROVED** [review]
LGTM

**Pull request merged**